src/dotnet/APIView/APIViewWeb/wwwroot/js/site.js

```diff
@@ -4,6 +4,43 @@
 // Write your Javascript code.
 $(function () {
     let commentFormTemplate = $("#comment-form-template");
+    attachEventHandlers(document);
+
+    function attachEventHandlers(element, id=null) {
+        let thisRow = $(document.getElementById(id)).parents(".code-line").first();
+
+        $(element).find(".commentable").off().click(function () {
```

Comment:
You would never need to resubscribe to this and `line-comment-button` clicks. Move them out.

Reply:
You wouldn't need `off()` too.

Reply:
The `off()` got left in by accident while I was debugging an issue with erroneous comment submissions because of `comment-submit-button` events being attached too many times. Thanks for the catch!